Resources/Prototypes/Entities/Structures/Furniture/toilet.yml

```diff
@@ -51,6 +51,15 @@
       disposals: !type:Container
   - type: Physics
     bodyType: Static
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.4
+        density: 190
+        mask:
+        - TableMask
   - type: Construction
```

Comment:
Why is it using tablemask and why does it not have a layer?

Reply:
> Why is it using tablemask and why does it not have a layer?

The fixture was copied from `SeatBase` prototype. It doesn't have mask neither.

Reply:
We can't go through the toilet with any of layers 'cause mob mask has `Impassable | HighImpassable | MidImpassable | LowImpassable` groups. So we don't need to have collision with any mob.
Mask obviously should be just MachineMask